scss/_custom-forms.scss

```diff
@@ -311,6 +311,12 @@
 
   &:focus {
     outline: none;
+
+    // Pseudo-elements must be split across multiple rulesets to have an affect.
+    // No box-shadow() mixin for focus accessibility.
+    &::-webkit-slider-thumb { box-shadow: $custom-range-thumb-focus-box-shadow; }
```

Comment:
@ysds Doesn't autoprefixer take care of the prefixes?

Reply:
Not in the selectors, no. All these need to be split as the comment says or no browser picks it up. Strange, but it's common across most of these form styles.

Reply:
As @mdo said. Also I found @ai's comment in https://github.com/postcss/autoprefixer/issues/279

> Main problem here, that we absolutly didn’t know unprefixed version of this selectors. Autoprefixer is developed to be able to remove Autoprefixer and future and your CSS will be worked.

> Try PostCSS and create polyfill.

Reply:
Autoprefixer can add prefixes in selectors (like it support `:: placeholder`). But Autoprefixer need to have W3C draft to have unprefixed version.

Reply:
Yeah, I was mostly wondering :)